src/pages/index.tsx

```diff
@@ -75,8 +75,8 @@ function Memes({ memeFromId }: { memeFromId?: MemesProps }) {
 
   const tags = [
     {
-      label: "Memepalooza 6",
-      value: "memepalooza 6",
+      label: "Memepalooza 7",
+      value: "memepalooza 7",
     },
     {
```

Comment:
we should bring back the default categories in the next version. It is easier for people to tag memes that way.